extensions/amp-video/validator-amp-video.protoascii

```diff
@@ -56,6 +56,10 @@ tags: {  # <amp-video>
   attrs: { name: "placeholder" }
   attrs: { name: "poster" }
   attrs: {
+    name: "crossorigin"
+    value_regex: "(^$|anonymous|use-credentials)"
```

Comment:
since we are propagating this down to `<video>` without doing anything with it, I wouldn't even validate the values as standards may evolve. (we don't validate values for `crossorigin` on `link` either)